### DISCUSSION TITLE
GitHub Certificationsについて

### DISCUSSION THREAD

**Discussion by koheihida** [community/community#145117] [New to GitHub]
ご担当者様

GitHub Certificationsについて質問させていただけますでしょうか。
認定試験の受験を検討しているのですが、
私は日本人のため、英語での受験は試験内容以上に難易度が高いです。
Google翻訳などの機能を使用して受験することは可能でしょうか。

ご確認よろしくお願いいたします。

**Comment by DevHanza** [comment]
Unfortunately, As I know, using translation tools during the exam is strictly prohibited. GitHub's certification exams are designed to assess your understanding of GitHub concepts and practices in English.

I recommend reaching out to GitHub's support team directly. They may be able to provide specific guidance or accommodations.

https://docs.github.com/en/support/contacting-github-support

**Reply by ghostinhershell** [reply]
Hey @DevHanza, thanks so much for jumping in and lending a helping hand ✋🏾

To follow-up and add to this, @koheihida, you can actually reach out to the team that handles GitHub certifications directly using this email address: [gh-certification@github.com](mailto:gh-certification@github.com). Good luck!

**Reply by koheihida** [reply]
＠DevHanza　＠ghostinhershell
Thank you! I’ll also contact the team in charge and do my best with my studies!!

**Reply by ghostinhershell** [reply]
Good luck!!! 🍀
